### PR TITLE
Feature/package info

### PR DESCRIPTION
**Pull request by nolik**
add raur client and modify info command.

### CODE REVIEW

**Comment by Morganamilo** [comment]
I have my own branch of ruar but the maintainer seems to be mia so I can't get it merged. Anyway you may want to consider depping on mybranch.

**Comment by nolik** [comment]
> I have my own branch of ruar but the maintainer seems to be mia so I can't get it merged. Anyway you may want to consider depping on mybranch.
what's your branch name, and what are you focus on >?

**Comment by Morganamilo** [comment]
See my open PR and related issue here: https://gitlab.com/davidbittner/raur/merge_requests/3

File: src/main.rs [hunk 1]
```diff
@@ -191,11 +192,11 @@ fn main() {
...
 		}
 		CliArgs::Search { target } => {
-			eprintln!(
-				"Results for '{}', sorted by popularity: \
-				 https://aur.archlinux.org/packages/?K={}&SB=p&SO=d",
-				target, target
-			);
+			let result = search(target, SearchStrategy::Name);
+			match result {
+				Ok(raur_result) => println!("Search result: '{:?}'", raur_result.packages),
```
Comment (vn971) [inline src/main.rs]: Just tried that. We definitely need to improve it. I'm thinking of the suggestion made here: https://github.com/vn971/rua/issues/13 on using `prettytable`. Seems a good fit to us?

Comment (vn971) [reply src/main.rs]: For reference, the current output is:
```
rua search rua
Search result: '[Package { id: 290271, name: "python-ruamel.base", package_base_id: Some(109615), package_base: Some("python-ruamel.base"), version: "1.0.0-1", description: "Common routines for ruamel Python packages", url: Some("https://bitbucket.org/ruamel/base"), num_votes: 0, popularity: 0.0, out_of_date: None, maintainer: Some("SpotlightKid"), first_submitted: Some(1459620075), last_modified: Some(1459621349), url_path: "/cgit/aur.git/snapshot/python-ruamel.base.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }, Package { id: 419628, name: "python-ruamel-yaml-hg", package_base_id: Some(123320), package_base: Some("python-ruamel-yaml-hg"), version: "0.15.6.r1.498e40e91a0c-1", description: "YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order", url: Some("https://bitbucket.org/ruamel/yaml"), num_votes: 2, popularity: 0.0, out_of_date: None, maintainer: Some("petronny"), first_submitted: Some(1497363852), last_modified: Some(1497410399), url_path: "/cgit/aur.git/snapshot/python-ruamel-yaml-hg.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }, Package { id: 538958, name: "python2-ruamel.ordereddict", package_base_id: Some(135455), package_base: Some("python2-ruamel.ordereddict"), version: "r41.983a529c0f41-1", description: "This is an implementation of an ordered dictionary with Key Insertion Order", url: Some("https://pypi.org/project/ruamel.ordereddict"), num_votes: 0, popularity: 0.0, out_of_date: None, maintainer: None, first_submitted: Some(1535438507), last_modified: Some(1535438507), url_path: "/cgit/aur.git/snapshot/python2-ruamel.ordereddict.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }, Package { id: 576079, name: "python-ruamel-yaml0.15.51", package_base_id: Some(138711), package_base: Some("python-ruamel-yaml0.15.51"), version: "0.15.51-1", description: "YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order", url: Some("https://bitbucket.org/ruamel/yaml"), num_votes: 0, popularity: 0.0, out_of_date: None, maintainer: Some("rafaelff"), first_submitted: Some(1547295869), last_modified: Some(1547295869), url_path: "/cgit/aur.git/snapshot/python-ruamel-yaml0.15.51.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }, Package { id: 502085, name: "python2-ruamel-yaml", package_base_id: Some(127675), package_base: Some("python2-ruamel-yaml"), version: "0.15.37-1", description: "YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order", url: Some("https://bitbucket.org/ruamel/yaml"), num_votes: 1, popularity: 0.009582, out_of_date: Some(1534324022), maintainer: None, first_submitted: Some(1512205983), last_modified: Some(1523436092), url_path: "/cgit/aur.git/snapshot/python2-ruamel-yaml.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }, Package { id: 604630, name: "rua", package_base_id: Some(137004), package_base: Some("rua"), version: "0.13.2-1", description: "secure AUR helper written in Rust", url: Some("https://github.com/vn971/rua"), num_votes: 3, popularity: 1.607477, out_of_date: None, maintainer: Some("vasya"), first_submitted: Some(1540823201), last_modified: Some(1555495709), url_path: "/cgit/aur.git/snapshot/rua.tar.gz", dependencies: None, make_dependencies: None, license: None, keywords: None }]'
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /target
+/Cargo.lock
```
[the remark this answers is not in the document]
Comment (nolik) [reply .gitignore]: thx, don't n about it.

File: src/main.rs (hunk 1 above)
Comment (nolik) [reply src/main.rs]: will try to add pretty table.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 /target
+.idea/
```
Comment (vn971) [inline .gitignore]: BTW, this is often placed in user-s config as well, not project-s. After all, we can't really always include ALL of the editors to each project, right? Try moving the ignore line to `~/.config/git/ignore`, I think it'll work. Or if you're on Windows, https://help.github.com/en/articles/ignoring-files

Comment (nolik) [reply .gitignore]: y

File: src/main.rs (hunk 1 above)
Comment (nolik) [reply src/main.rs]: `+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| Id     | Name                       | Version                  | Description                                                                                                 |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 290271 | python-ruamel.base         | 1.0.0-1                  | Common routines for ruamel Python packages                                                                  |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 419628 | python-ruamel-yaml-hg      | 0.15.6.r1.498e40e91a0c-1 | YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 538958 | python2-ruamel.ordereddict | r41.983a529c0f41-1       | This is an implementation of an ordered dictionary with Key Insertion Order                                 |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 576079 | python-ruamel-yaml0.15.51  | 0.15.51-1                | YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 502085 | python2-ruamel-yaml        | 0.15.37-1                | YAML parser/emitter that supports roundtrip preservation of comments, seq/map flow style, and map key order |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| 604630 | rua                        | 0.13.2-1                 | secure AUR helper written in Rust                                                                           |
+--------+----------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
`

**Comment by vn971** [comment]
Merged. Thanks for the contribution!

I changed a few things before merging:
* fixed mixed tabs and spaces
* fixed the title to be a title (not row), which in turn allows a bit nicer formatting
* minimised formatting so that it takes fewer space in terminal
* alpabetically sorted things that could be sorted

And took the rest.:)

**Pull request merged**